script/Arcueid/Day 5/05_00_ARC05_5.txt

```diff
@@ -648,7 +648,7 @@ The wrath of nature that crushes it.
 [sha:0cbd0216accdb72d6cea8f400853a48dee602b4c]{
 -- Page 419, Offset 9693.
 -- その体に備わった当然の権利である。
-Of course such a body would possess these rights.
+Of course, such a body would possess these rights.
```

Comment:
Revert. Of course with a comma feels like a you are saying something that is counter-flow to the rest of the conversation, but this is "of _course_ such a body would possess these rights"